doc/reference/tlspolicy.md

```diff
@@ -15,7 +15,7 @@
 
 | **Field**              | **Type**                                                                                                                                     | **Required** | **Description**                                                                                                                                  |
 |------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|:------------:|--------------------------------------------------------------------------------------------------------------------------------------------------|
-| `targetRef`            | [Gateway API LocalPolicyTargetReference](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.LocalPolicyTargetReference)              |     Yes      | Reference to a Kuberentes resource that the policy attaches to                                                                                   |
```

Comment:
@KevFan should this be updated to reference sectionName like the DNS & RateLimit policies?
It looks like support landed in https://github.com/Kuadrant/kuadrant-operator/pull/983

Reply:
Updated :+1: